### ISSUE TITLE
Backup support for raw File Format

### ISSUE THREAD

**Issue by madmax01** [abbbi/virtnbdbackup#61]
Hello,

for opennebula VMs they use an hda which is raw for accessing contextualization File

is there an way virtnbdbackup can handle all "Images" based on an VM?   raw + qcow2

if restoring the disk.0,......then disk.1 still missing to get the VM operativ

thx

Max

**Comment by abbbi** [comment]
see README: https://github.com/abbbi/virtnbdbackup#supported-disk-formats--raw-disks

**Comment by madmax01** [comment]
Hello,

i sure tried this out.

virtnbdbackup  -d test-43  -l full -r -z -o /tmp/test/

but hda is getting skipped 

"getDomainDisks [MainThread]: Skipping attached CDROM / Floppy: [hda]"

**works**
qemu-img info disk.0
image: disk.0
file format: qcow2

**skipped**
qemu-img info disk.1
image: disk.1
file format: raw
virtual size: 364 KiB (372736 bytes)
disk size: 364 KiB

**Comment by abbbi** [comment]
> "getDomainDisks [MainThread]: Skipping attached CDROM / Floppy: [hda]"
> **skipped** qemu-img info disk.1 image: disk.1 file format: raw virtual size: 364 KiB (372736 bytes) disk size: 364 KiB

yes, because this is a type floppy or CDROM image, which virtnbdbackup does skip by default.
I understand this might be an issue on opennebula where there is cloud-init related stuff on the cdrom image, but
currently there is no option to forcefully include cdrom or floppy type images in the backup too.
Pull requests welcome.
